x-pack/plugins/maps/public/shared/layers/styles/components/vector/size/size_range_selector.js

```diff
@@ -9,66 +9,42 @@ import PropTypes from 'prop-types';
 import {
   EuiFormRow,
   EuiFlexGroup,
-  EuiFlexItem,
+  EuiFlexItem
 } from '@elastic/eui';
-import { ValidatedRange } from '../../../../../components/validated_range';
+import { ValidatedDualRange } from 'ui/validated_range';
 import { i18n } from '@kbn/i18n';
 import { DEFAULT_MIN_SIZE, DEFAULT_MAX_SIZE } from '../../../vector_style_defaults';
 
+
 export function SizeRangeSelector({ minSize, maxSize, onChange }) {
 
-  const onSizeChange = (min, max) => {
+  const onSizeChange = ([min, max]) => {
     onChange({
-      minSize: min,
-      maxSize: max
+      minSize: Math.max(DEFAULT_MIN_SIZE, parseInt(min, 10)),
+      maxSize: Math.min(DEFAULT_MAX_SIZE, parseInt(max, 10))
     });
   };
 
-  const onMinSizeChange = (updatedMinSize) => {
-    onSizeChange(updatedMinSize, updatedMinSize > maxSize ? updatedMinSize : maxSize);
-  };
-
-  const onMaxSizeChange = (updatedMaxSize) => {
-    onSizeChange(updatedMaxSize < minSize ? updatedMaxSize : minSize, updatedMaxSize);
-  };
-
   return (
     <EuiFlexGroup>
       <EuiFlexItem>
         <EuiFormRow
           label={
-            i18n.translate('xpack.maps.styles.vector.size.minLabel', {
-              defaultMessage: 'Min size'
-            })
-          }
-          compressed
-        >
-          <ValidatedRange
-            min={DEFAULT_MIN_SIZE}
-            max={DEFAULT_MAX_SIZE}
-            value={minSize}
-            onChange={onMinSizeChange}
-            showInput
-            showRange
-          />
-        </EuiFormRow>
-      </EuiFlexItem>
-      <EuiFlexItem>
-        <EuiFormRow
-          label={
-            i18n.translate('xpack.maps.styles.vector.size.maxLabel', {
-              defaultMessage: 'Max size'
+            i18n.translate('xpack.maps.styles.vector.size.minMaxLabel', {
```

Comment:
I don't think any label is needed. The style already has a label so the slider does not need its own
<img width="433" alt="Screen Shot 2019-03-14 at 10 48 41 AM" src="https://user-images.githubusercontent.com/373691/54375494-c0559500-4646-11e9-8fe8-35e24cbe81d9.png">